modules/ROOT/pages/amazon/amazon-dynamodb-connector.adoc

```diff
@@ -8,298 +8,58 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
+Anypoint Connector for Amazon DynamoDB (Amazon DynamoDB Connector) provides connectivity to the Amazon DynamoDB API. Amazon DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability.
 
+For compatibility information and fixed issues, see the xref:release-notes::connector/amazon-dynamodb-connector-release-notes-mule-4.adoc[Amazon DynamoDB Connector Release Notes]
 
-Amazon DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability.
+== Common Use Cases For the Connector
 
-Release Notes: xref:release-notes::connector/amazon-dynamodb-connector-release-notes-mule-4.adoc[Amazon DynamoDB Connector Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-dynamodb-connector/[Amazon DynamoDB Connector]
+Amazon DynamoDB Connector enables you to interact with Amazon DynamoDB to create a database table that can:
 
-== Prerequisites
-
-To use this information, you must be familiar with Amazon DynamoDB API, Mule, Anypoint Connectors, Anypoint Studio, Mule
-concepts, elements in a Mule flow, and Global Elements.
-
-For software requirements and compatibility information, see the Connector Release Notes.
-
-To use this connector with Maven, view the pom.xml dependency information in
-the Dependency Snippets in Anypoint Exchange.
-
-To use the Amazon DynamoDB Connector, you must have the following:
-
-- Access to Amazon Web Services - DynamoDB
-- AWS Identity and Access Management (IAM) user account credentials to access AWS with the connector
-- Anypoint Studio version 7.0 or later, or Design Center
-
-== POM File Information
-
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>com.mulesoft.connectors</groupId>
-  <artifactId>mule-amazon-dynamodb-connector</artifactId>
-  <version>RELEASE</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
-
-Mule converts `RELEASE` to the latest version. To specify a version, view 
-https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-dynamodb-connector/[Amazon DynamoDB Connector]
-in Anypoint Exchange and click *Dependency Snippets*.
-
-== Add the Connector to a Studio Project
-
-Anypoint Studio provides two ways to add the connector to your Studio project: 
-
-* From the Exchange button in the Studio taskbar
-* From the Mule Palette view
+* Store and retrieve any amount of data
+* Serve any level of request traffic
+* Automatically distribute the data and traffic for the table over a sufficient number of servers to handle the request capacity and the amount of data stored. Amazon Dynamo DB Connector does this while maintaining consistent, fast performance.
```

Comment:
Two solutions for the two-sentence structure:
Per our style ref....
```suggestion
* Automatically distribute the data and traffic for the table over a sufficient number of servers to manage the request capacity and the amount of data stored. +
Amazon Dynamo DB Connector does this while maintaining consistent, fast performance.
```
OR rewording for parallelism....
```suggestion
* Automatically distribute the data and traffic for the table over a sufficient number of servers to manage the request capacity and the amount of data stored this while maintaining consistent, fast performance
```

Reply:
One other suggestion, or really a question: is the third list item really sort of a recap of the first two?
If so, that might suggest other presentation possibilities.

Reply:
Just a note about "handle":
I think it's a bit vague, and not sure how translators "handle" it (see what I did there?).
I know that most sentence contexts imply the precise meaning of this term (in the previous sentence, for example, "handle" = "translate" or "localize" or similar).

However, I think it would be best if we simply use the precise meaning in context: manage, process, prioritize, control....

If you look this up, the first verb use is all about touching, followed by managing and then by carrying as merchandise (the noun entries are for, first, something to grip and, second, a nickname). Kinda' interesting.

https://www.thesaurus.com/browse/handle